pytest_fixtures/component/satellite_auth.py

```diff
@@ -271,7 +272,7 @@ def enroll_configure_rhsso_external_auth(module_target_sat):
         'yum -y --disableplugin=foreman-protector install '
         'mod_auth_openidc keycloak-httpd-client-install'
     )
-    if module_target_sat.os_version.major == '8':
+    if module_target_sat.os_version.major == 8:
```

Comment:
We shouldn't handle such conditions for 6.12 since el8 is the default

Reply:
thanks for pointing out !

Reply:
btw concerning the change made - best way to be safe here is to force type of the variable, str or int doesn't matter, but match the type of the scalar being compared.

```
if int(sat.os_version.major) == 8
```